macaroonbakery/keys.py

```diff
@@ -32,20 +46,31 @@ def __eq__(self, other):
 
 
 class PublicKey(object):
+    ''' A 256-bit Ed25519 public key.
```

Comment:
A public key used by the bakery to encrypt third party caveats.
Every discharger is associated with a public key which is
used to encrypt third party caveat ids addressed to that discharger.
Internally, it is a 256 bit Ed25519 public key.

?